### PR TITLE
Update docs with host.json options

### PR DESCRIPTION
**Pull request by MaddyDev**
Updated the instructions to redirect to how to use host.json options and also added note that we currently override app settings over host.json.

### CODE REVIEW

**Review by JetterMcTedder — CHANGES_REQUESTED** [review]
Can we maybe rephrase this?
**NOTE:** We currently override configuration from app settings over host.json.

We are stating that we take the configuration in app settings and over write/use these instead of the ones in the host.json if present?

Maybe something like:

**NOTE:** Currently, SQL trigger bindings configurations from app settings will take precedence over settings in the host.json file.

**Review by JetterMcTedder — APPROVED** [review]

**Review by Charles-Gagnon — APPROVED** [review]

**Review by lucyzhang929 — APPROVED** [review]

**Comment by MaddyDev** [comment]
Merging this in sine the test failure isn't related to the changes in the PR.
